src/edit/domchange.js

```diff
@@ -35,7 +35,7 @@ function parseBetween(pm, from, to) {
     if (next.nodeType != 1 || !next.hasAttribute("pm-offset")) ++endOff
     else break
   }
-  let domSel = window.getSelection(), find = null
+  let domSel = pm.root.getSelection(), find = null
```

Comment:
Can this code be executed when `pm.root` is `null`? If so a null check may be worthwhile.